CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,86 @@
+## Goal
+
+Our goal is to provide a space where it is safe for everyone to contribute to
+open-source software in a respectful and cooperative manner.
+
+We value all contributions and want to make this organization and its surrounding community
+a place for everyone.
+
+As members, contributors and everyone else who may participate in the development, we strive
+to keep the entire experience civil.
+
+## Standards
+
+Our community standards exist in order to make sure everyone feels comfortable contributing
+to the project(s) together.
+
+Our standards are:
+ - Do not harass, attack, or in any other way discriminate against anyone, including for their protected traits, including, but not limited to, sex, religion, race, appearance, gender, identity, nationality, sexuality, etc.
+ - Do not go off-topic, do not post spam.
+ - Treat everyone with respect.
+
+Examples of breaking each rule respectively include:
+ - Harassment, bullying or inappropriate jokes about another person.
+ - Posting distasteful imagery, trolling, or political discussions.
+ - Treating someone as worse because of their lack of understanding of an issue.
+
+## Enforcement
+
+Enforcement of this CoC is done by the members of the hyprwm organization.
```

Comment:
as we are the only members of the org :)

Reply:
Ah, perhaps clarify the "hyprwm GitHub organization"?

You might also want to define a conduct enforcement team here as having this specific set of members, separately from the GitHub org, so that the two teams can be distinct. You might not want someone who joins the GitHub org to necessarily be responsible for conduct enforcement, or vice versa.